src/currency/symbol-properties.js

```diff
@@ -13,7 +14,8 @@ return function( currency, cldr, options ) {
 	var currencySpacing, pattern, symbol, symbolEntries,
 		regexp = {
 			"[:digit:]": /\d/,
-			"[:^S:]": regexpNotS
+			"[:^S:]": regexpNotS,
+			"[[:^S:]&[:^Z:]]": regexpNotSAndZ
```

Comment:
Had a question: in order to actually test this in the `symbol-properties.js` I'd need to install the CLDR v38. At the same time I realize that we need to support CLDR versions prior to v38. Any suggestions on how to do that nicely? So far I've just written a unit test for that particular regex.

Reply:
A unit test for that particular regex is fine :+1:. Ideally, we'd run our tests using a matrix of CLDR versions, but assuming new versions of CLDR are backward compatible, using the latest is reasonable.

Reply:
Err, didn't understand this one. Should I update the CLDR references here up to the latest version?